client/templates/HomeView.ejs

```diff
@@ -127,6 +128,14 @@
             margin-right:32px;
         }
 
+        #content-middle {
+            margin-bottom: 3px;
```

Comment:
this should be either 2px or 4px so we always size things as 2, 4, 8, 12, 16, 32 or 64